Sources/PackageMetadata/PackageMetadata.swift

```diff
@@ -189,18 +195,22 @@ public struct PackageSearchClient {
             self.indexAndCollections.findPackages(query) { result in
                 do {
                     let packages = try result.get().items.map {
-                        Package(
+                        let versions = $0.package.versions.sorted(by: >)
+                        let latestVersion = versions.first
+                        
+                        return Package(
                             identity: $0.package.identity,
                             location: $0.package.location,
                             versions: $0.package.versions.map(\.version),
-                            licenseURL: nil,
+                            licenseURL: $0.package.license?.url,
                             readmeURL: $0.package.readmeURL,
                             repositoryURLs: nil,
                             resources: [],
-                            author: nil,
-                            description: nil,
-                            publishedAt: nil,
-                            latestVersion: nil,
+                            author: latestVersion?.author.map { .init($0) },
+                            description: latestVersion?.summary,
+                            publishedAt: latestVersion?.createdAt,
+                            signingEntity: latestVersion?.signer.map { SigningEntity(signer: $0) },
+                            latestVersion: latestVersion?.version,
```

Comment:
I think we can set some of these using the package's latest version listed in the collection/index result.